src/lib.rs

```diff
@@ -103,13 +103,56 @@ pub struct CmdArgs {
   /// If set, make sure to set --federate-process-index differently for each.
   #[arg(long, default_value_t = 1, env = "LEMMY_FEDERATE_PROCESS_COUNT")]
   federate_process_count: i32,
+  #[command(subcommand)]
+  subcommand: Option<CmdSubcommand>,
+}
+
+#[derive(Subcommand, Debug)]
+enum CmdSubcommand {
+  /// Do something with migrations, then exit.
+  Migration {
+    #[command(subcommand)]
+    subcommand: MigrationSubcommand,
+    #[arg(long, default_value_t = false)]
+    all: bool,
+    #[arg(long, default_value_t = 1)]
+    number: u64,
+  },
+}
```

Comment:
These params need documentation, especially number is unclear.

Reply:
done